### DISCUSSION TITLE
Is there a Copilot.vim changelog?

### DISCUSSION THREAD

**Discussion by Yuki-cpp** [community/community#64798] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hello :slightly_smiling_face: 

I'm using Copilot with Neovim and as far as the product goes, no issues.

However, the commits on the copilot plugin repository are all releases commits with a simple "Copilot.vim X.Y.Z" title and no description. This makes following up with the changes pretty hard.

It seems that the VSCode version has a changelog (as shown in [this answer](https://github.com/orgs/community/discussions/61580#discussioncomment-6528924)), is there one somewhere for the Neovim version? 
If no, could adding one be considered?

Thank you

---
Output of my `:Copilot version` if it helps.
```
copilot.vim 1.10.1
Neovim 0.10.0-dev-923+g6462ee1c1
dist/agent.js 1.105.350
Node.js 20.0.0
```

**Comment by Akash1134** [comment, marked as answer]
Hi @Yuki-cpp ,

Currently we don't have a detailed changelog for NeoVim but I've shared your feedback with the relevant team.

Thanks!

**Reply by Yuki-cpp** [reply]
Thanks

**Reply by ViViDboarder** [reply]
This has become an issue for me lately as it seems like a breaking change was introduced (min version of node from 18 to 20), so I would like to pin the version, however I needed to do some sleuthing to figure out what and when it changed. I'm pretty sure I've found it at `1.43`, but some warning would have been helpful.

**Reply by yermulnik** [reply]
Yep, only can track changes based on commits tracked using Atom feed for Git tags 😕

**Comment by a-h** [comment]
I'm still confused by this. I'm currently on copilot.vim version v1.34.0. The current version is v1.41.0 - so what's the difference?

Absolutely impossible to know - the commit comments are just version numbers.

<img width="893" alt="Screenshot 2024-12-02 at 17 20 17" src="https://github.com/user-attachments/assets/414de052-2a79-400c-9885-170789009cd3">

And there's so many code changes, I can't make sense of the intention.

<img width="1507" alt="Screenshot 2024-12-02 at 17 20 55" src="https://github.com/user-attachments/assets/9684c442-93f3-476c-938e-ce1141cbe784">

Without a changelog, I don't know how I'm supposed to know what new features are included.

**Comment by a-h** [comment]
Was hoping to find that the Chat feature was now included in the Vim plugin as per https://code.visualstudio.com/docs/copilot/getting-started-chat but alas, doesn't look like it.
